docker-wrapper.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+# Variables setting
+HOST=`hostname -i`
+petalscli="/opt/petals-deployer/bin/petals-cli.sh -h $ENV_CLI_HOST -n $ENV_CLI_PORT -u $ENV_CLI_USER -p $ENV_CLI_PASS -y -c "
```

Comment:
Environement variables are set when execute `docker run...`

Reply:
They are. They have a default value set in the Dockerfile, and can be overridden in the `docker run`.
What is your point ?